kubernetes/client/rest.py

```diff
@@ -39,10 +39,7 @@ def __init__(self, resp):
         self.reason = resp.reason
         self.data = resp.data
 
-    def getheaders(self):
-        """Returns a dictionary of the response headers."""
-        return self.urllib3_response.getheaders()
```

Comment:
One alternative is to restrict the version of urllib3 that this client can support (< 2.0.0). That doesn't seem to be a good user experience. I'm not sure what's the adoption rate of `urllib3 >= 2.0.0` as of today. cc @yliaog

Reply:
Never mind. 2.0.0 has been released for over a year: https://pypi.org/project/urllib3/#history